includes/class-wc-ajax.php

```diff
@@ -1294,7 +1294,7 @@ public static function json_search_products( $term = '', $include_variations = f
 			$ids = array_slice( $ids, 0, absint( $_GET['limit'] ) );
 		}
 
-		$product_objects = array_filter( array_map( 'wc_get_product', $ids ), 'wc_products_array_filter_editable' );
+		$product_objects = array_filter( array_map( 'wc_get_product', $ids ), 'wc_products_array_filter_visible' );
```

Comment:
This line should not be included.

We should be able to link them. We just need to ensure they are not displayed until visible on the frontend.

Reply:
Reverted to use the original filter.